home.html

```diff
@@ -0,0 +1,327 @@
+            <li>Login</li>
+            <li class="signup-btn">Sign up</li>
+            <li class="language-dropdown" style="font-size: smaller">
+              <span>EN&#9660;</span>
+              <ul class="dropdown-content">
+                <li>Hindi</li>
+                <li>German</li>
+              </ul>
+            </li>
+          </ul>
+        </section>
+      </nav>
+      <main>
+        <span class="left-section">
+          <h5>BEST DESTINATIONS AROUND THE WORLD</h5>
+          <h1>
+            Travel, enjoy <br />and live a new <br />
+            and full life
+          </h1>
+          <p>
+            Built Wicket Longer admire do barton vanity itself do in it. <br />
+            Preferred to sportsmen it engrossed listening. Park gate <br />
+            sell they west hard for the.
+          </p>
+          <button class="cta-button">Find out more</button>
+        </span>
+        <span class="right-section">
+          <img src="elements\traveller.png" alt="" />
+        </span>
+      </main>
+    </header>
+    <section class="category">
+      <div>
+        <header>CATEGORY</header>
+        <section class="category-content">We Offer Best Services</section>
+      </div>
+      <img
+        src="elements\plus.png"
+        alt=""
+        style="position: absolute; transform: translate(645px, 20%)"
+      />
+    </section>
+
+    <div class="services-list">
+      <div class="service-card">
+        <div class="card-img">
+          <img src="elements\weather.png" alt="Weather Image" />
+        </div>
+
+        <div class="card-content">
+          <h2>Calculated Weather</h2>
+          <p>Built Wicket longer admire do barton vanity itself do in it.</p>
+        </div>
+      </div>
+      <div class="service-card">
+        <div class="card-img">
+          <img src="elements\flight.png" alt="Flight Image" />
+        </div>
+        <div class="card-content">
+          <h2>Best Flights</h2>
+          <p>Engrossed listening. Park gate sell they west hard for the.</p>
+        </div>
+      </div>
+      <div class="service-card">
+        <div class="card-img">
+          <img src="elements\event.png" alt="Event Image" />
+        </div>
+        <div class="card-content">
+          <h2>Local Events</h2>
+          <p>
+            Barton vanity itself do in it. Preferred to men it engrossed
+            listening.
+          </p>
+        </div>
+      </div>
+      <div class="service-card">
+        <div class="card-img">
+          <img src="elements\customization.png" alt="Customization Image" />
+        </div>
+        <div class="card-content">
+          <h2>Customization</h2>
+          <p>We deliver outsourced aviation services for military customers.</p>
+        </div>
+      </div>
+    </div>
+    <section class="top-selling">
+      <header>
+        <div><h4>Top selling</h4></div>
+        <div><h1>Top Destinations</h1></div>
+      </header>
+      <main class="top-selling-list">

```

Comment:
The ```<header>``` element has an identical meaning to the site-wide banner landmark role, unless nested within sectioning content. Then, the <header> element is not a landmark. The <header> element can define a global site header, described as a banner in the accessibility tree.

Reply:
resolved